letsdrill.rb

```diff
@@ -1,24 +1,34 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+
+  if integer.between?(90,100)
+    "A"
+  elsif integer.between?(80,90)
+    "B"
+  elsif integer.between?(70,80)
+    "C"
+  elsif integer.between?(60,70)
+    "D"
+  elsif
+    "F"
+  end
 
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  array.sort_by(&:length).first
```

Comment:
cool, use of Symbol#to_proc! 

__pro-tip:__ `.min_by` will let you drop your use of `.first` and make this even more concise.